lib/api/controllers/documentController.js

```diff
@@ -37,30 +37,30 @@ const { dumpCollectionDocuments } = require('../../util/dump-collection');
  * @value event type (empty for actions that are not linked to events)
  */
 const actions = {
-  'count': '',
-  'create': 'write',
-  'createOrReplace': 'write',
-  'delete': 'delete',
-  'deleteByQuery': 'delete',
-  'deleteFields': '',
-  'exists': 'get',
-  'export': 'get',
-  'get': 'get',
-  'mCreate': 'write',
-  'mCreateOrReplace': 'write',
-  'mDelete': 'delete',
-  'mExists': 'get',
-  'mGet': 'get',
-  'mReplace': 'write',
-  'mUpdate': 'update',
-  'mUpsert': '',
-  'replace': 'write',
-  'scroll': '',
-  'search': 'get',
-  'update': 'update',
-  'updateByQuery': 'update',
-  'upsert': 'update',
-  'validate': '',
+  count: '',
+  create: 'write',
+  createOrReplace: 'write',
+  delete: 'delete',
+  deleteByQuery: 'delete',
+  deleteFields: 'update',
+  exists: 'get',
+  export: 'get',
+  get: 'get',
+  mCreate: 'write',
+  mCreateOrReplace: 'write',
+  mDelete: 'delete',
+  mExists: 'get',
+  mGet: 'get',
+  mReplace: 'write',
+  mUpdate: 'update',
+  mUpsert: '',
```

Comment:
This should also trigger the update generic event

Reply:
Are you sure you are reviewing the right commit? because mUpsert triggers a generic update event.
`mUpsert: 'update',`